living_documentation_regime/living_documentation_generator.py

```diff
@@ -298,26 +302,33 @@ def _generate_markdown_pages(self, issues: dict[str, ConsolidatedIssue]) -> None
             index_org_level_template,
             index_repo_page_template,
             index_data_level_template,
+            report_page_template,
         ) = self._load_all_templates()
 
         # Generate a markdown page for every issue
         for consolidated_issue in issues.values():
             self._generate_md_issue_page(issue_page_detail_template, consolidated_issue)
+            if consolidated_issue.errors:
+                for error_type, error_message in consolidated_issue.errors.items():
+                    repository_id: str = consolidated_issue.repository_id
+                    number: int = consolidated_issue.number
```

Comment:
These two lines can be outside of for, right?

Reply:
Great catch: TBD